pages/linux/pihole.md

```diff
@@ -0,0 +1,28 @@
+# pihole
+
+> Terminal interface for the Pi-Hole ad-blocking DNS server.
+> <https://pi-hole.net>.
+
+- Check status:
```

Comment:
What does this check the status of?

Reply:
The Pi-hole daemon. Pi-hole is an ad-blocking DNS server (meant to run on a raspi, can run on any linux machine), this command tells you if it's up and running or not. Its output looks like this:

```
╭─ robert@cloudBucket  ~  
╰─ pihole status
  [✓] DNS service is running
  [✓] Pi-hole blocking is Enabled
```

Reply:
Maybe it would be better as:
```suggestion
- Check the status of the daemon:
```

Reply:
I tweaked it. New commit inbound!